From dec0dd1be9db36131d7c7ee9c9061f3a7ef3b8c4 Mon Sep 17 00:00:00 2001
From: Michael Webster <miketwebster@gmail.com>
Date: Sat, 6 May 2017 09:10:47 -0400
Subject: [PATCH] st-bin.c: Fix y_fill setter (it was setting x_fill to the
 value of the existing y_fill, before it is set in the same call).

---
 src/st/st-bin.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: cinnamon/src/st/st-bin.c
===================================================================
--- cinnamon.orig/src/st/st-bin.c	2017-05-19 17:46:46.015080553 +0200
+++ cinnamon/src/st/st-bin.c	2017-05-19 17:46:46.011080426 +0200
@@ -252,7 +252,7 @@
 
     case PROP_Y_FILL:
       st_bin_set_fill (bin,
-                       bin->priv->y_fill,
+                       bin->priv->x_fill,
                        g_value_get_boolean (value));
       break;
 
